pandas/_libs/tslibs/period.pyx

```diff
@@ -290,6 +319,21 @@ cdef int dInfoCalc_SetFromAbsDateTime(date_info *dinfo,
     """
     Set the instance's value using the given date and time.
     Assumes GREGORIAN_CALENDAR.
+
+    Parameters
+    ----------
+    dinfo : date_info*
+    absdate : int64_t days elapsed since datetime(1, 1, 1)
+    abstime : double
+        seconds elapsed since beginning of day described by absdate
+
+    Returns
```

Comment:
this should no longer be needed. IOW this is the point of an errors declaration (and raising)

Reply:
I agree, will be removing the leftover-from-C bits (and camelCase) in the next pass.  For now this PR is focused on just the one bug.